python/katana/local/_graph.pyx

```diff
@@ -592,3 +602,12 @@ cdef class Graph(GraphBase):
     def global_in_degree(self, uint64_t node):
         # TODO(loc) needs shared-memory bi-directional view
         raise NotImplementedError()
+
+
+cdef class TxnContext:
+    @property
+    def __katana_address__(self):
+        """
+        Internal.
+        """
+        return <uint64_t>&self._txn_ctx
```

Comment:
Why was this needed?

Reply:
To enable the cython integration above.

Reply:
For `CythonReference` yes. If you need `MakeCythonWrapper` you also need to implement at least one of the `_make_from_address` methods.